core/spec/finders/spree/option_values/find_available_spec.rb

```diff
@@ -68,6 +68,23 @@ module Spree
           )
         end
       end
+
+      context 'ordering' do
```

Comment:
If I see correctly, there are no cases for sorting by option type position, only option values. At least it's not explicit. Could we expand on it to be sure?

Reply:
<img width="739" alt="Screenshot 2021-12-02 at 10 44 30" src="https://user-images.githubusercontent.com/55154/144397364-ebc92c23-6ba9-4f1c-a093-b317232cb10c.png">

@mad-eel you can sort option types in the Admin, and it should be respected later in the API / UI :)

Reply:
@damianlegawiec I think he means in the specs